README.md

```diff
@@ -27,7 +27,7 @@ or
 
 ## Usage
 
-```javascript
+```javaScript
```

Comment:
why?

Reply:
Force pushed to correct that incorrect change 3 seconds before you reviewed, I noticed it as well haha.